integration-tests/validation.py

```diff
@@ -75,13 +81,37 @@ def validate_status_messages(model):
     await action.wait()
     assert action.status == 'completed'
     for i in range(60):
-        resp = requests.get('http://' + action.data['results']['address'])
+        resp = await asyncify(requests.get)('http://' + action.data['results']['address'])
         if resp.ok:
             return
         await asyncio.sleep(1)
     raise MicrobotError('Microbot failed to start.')
 
 
+async def validate_dashboard(model, log_dir):
+    ''' Validate that the dashboard is operational '''
+    unit = model.applications['kubernetes-master'].units[0]
+    with NamedTemporaryFile() as f:
+        await unit.scp_from('config', f.name)
+        with open(f.name, 'r') as stream:
+            config = yaml.load(stream)
+    url = config['clusters'][0]['cluster']['server']
+    user = config['users'][0]['user']['username']
+    password = config['users'][0]['user']['password']
+    auth = requests.auth.HTTPBasicAuth(user, password)
+    resp = await asyncify(requests.get)(url, auth=auth, verify=False)
+    assert resp.ok is True
```

Comment:
Can you change this to `assert resp.status_code == 200`?

http://docs.python-requests.org/en/master/api/#requests.Response.ok